pkg/virtual/options/options.go

```diff
@@ -72,6 +76,11 @@ func (o *Options) NewVirtualWorkspaces(
 	wildcardKubeInformers kubernetesinformers.SharedInformerFactory,
 	wildcardKcpInformers kcpinformers.SharedInformerFactory,
 ) ([]rootapiserver.NamedVirtualWorkspace, error) {
+
```

Comment:
I think the main `kcp` process also calls this - if so, we'd want to make sure that we start the profiler only when we're in the standalone case.

Reply:
@stevekuznetsov I don't think we would be able to set the profiler address when it's getting called from `kcp` process but I could move the invocation to `cmd.go`.

Reply:
When we're running the virtual workspace server as part of `kcp`, we should be using the `kcp` options to run the (single) profiler server. When we're running standalone, we should use the built-in k8s APIServer profiling option for the VW to run it.

Reply:
I am not sure how to use the profiling option of APIServer and specify a particular address as it's config doesn't accept any endpoints as per  https://pkg.go.dev/k8s.io/apiserver/pkg/server?utm_source=gopls#Config.EnableProfiling
@sttts any pointers?

Reply:
yes, this belongs into cmd/virtual-workspaces/options.